lib/legacy_client/client.rb

```diff
@@ -97,6 +97,8 @@ def request(method, path, payload = nil, headers = {})
         headers[:content_type] = "application/json"
       end
 
+      headers = headers.merge(GdsApi::GovukHeaders.headers)
```

Comment:
Removed this in another commit. It's defaulted to {} everywhere and I can't see anywhere where headers is passed anything that could be nil.